components/Explorer/ExplorerPluginDetails.tsx

```
@@ -62,6 +63,53 @@ export function ExplorerPluginDetails({ plugins, type }: { plugins: PluginsList
+          </Stack>
+        </Fieldset>
+      ))}
+
+      {plugins.linkedAppDatas && plugins.linkedAppDatas.map((linkedAppData, idx) => (
+        <Fieldset key={idx} legend={<LabelTitle>Linked App Data</LabelTitle>}>
+          <Stack>
+            <AuthorityStat authority={linkedAppData.authority} name="Plugin" />
+            <AuthorityStat authority={linkedAppData.dataAuthority} name="Data" />
+          </Stack>
+        </Fieldset>
+      ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)